package.json

```diff
@@ -27,6 +27,7 @@
   },
   "scripts": {
     "build": "rollup -c",
+    "build-test": "rollup -c rollup.unit.config.js",
```

Comment:
Hmm, I think there is a way to only use one rollup.config.js and pass parameters.